Python/Image Requests/img.py

```diff
@@ -0,0 +1,41 @@
+import requests
+from PIL import Image
+def img_requests(txt):
+    response=requests.get("https://source.unsplash.com/random/{0}".format(txt))
+    file=open('sample_image.jpg','wb')
```

Comment:
Avoid using ambiguous filenames.

Reply:
actually i did not understand , could u give me an example for this